### PR TITLE
adds new application form

### PR DESCRIPTION
**Pull request by rmariani**
This one has an email autoresponder

### CODE REVIEW

File: content/community/hoonschool.md [hunk 1]
```diff
@@ -23,3 +23,5 @@ Hoon 101 is a six-week, introductory-level course. It’s appropriate for motiva
 We're in the process of designing advanced Hoon Courses, but they aren't ready yet. They'll cover advanced development with Hoon, such as app development, kernel development, and the details of the Hoon type system. Such classes will be appropriate for graduates of Hoon 101. Check back later if you're interested in enrolling.
 
 <script class="src="https://static.airtable.com/js/embed/embed_snippet_v1.js"></script><iframe id="apply" class="airtable-embed airtable-dynamic-height w-full" src="https://airtable.com/embed/shr8bb15QPR3hfQpG?backgroundColor=teal" frameborder="0" height="1300"></iframe>
```
Comment (matildepark) [inline content/community/hoonschool.md]: You should delete this one.

**Comment by rmariani** [comment]
oh duh

**Comment by rmariani** [comment]
deleted

**Comment by matildepark** [comment]
[It doesn't render.](https://deploy-preview-344--urbit-org.netlify.com/community/hoonschool/) I am assuming this is a permissions issue on the Google Form — it forces me to sign in to see it.

**Comment by rmariani** [comment]
I changed the permissions and it's rendering to me

**Comment by matildepark** [comment]
It does now! I'd just give the iFrame a height of 1500 so the full form is always visible — there's a scroll that isn't immediately apparent.

**Review by matildepark — APPROVED** [review]
Looks good.

**Pull request merged**